app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
     package="com.example.mayankjindal.voicecalculator">
 
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />+
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Please see that what impact this change would bring. 
I don't think that there is anything changed.